scripts/index_on_jsonb_field.txt

```diff
@@ -20,3 +29,29 @@ migration. See: https://its.cern.ch/jira/browse/INSPIR-375
 -- CREATE INDEX CONCURRENTLY json_ids_index ON records_metadata USING gin ((json -> 'ids'));
 
 -- DROP INDEX CONCURRENTLY json_ids_index;
+
+
+2. RECORDSMETADATA.JSON _export_to
+==================================
+The following SQL query adds an index to RecordMetadata.json field.
+The index can then be used in queries like:
+
+RecordMetadata.query.filter(RecordMetadata.json['_export_to'].op('@>')('[{"HAL": true}]'))
```

Comment:
This is exactly the query above, just expressed in a syntax closer to PostgreSQL. You can also write it as
```python
RecordMetadata.query.filter(type_coerce(RecordMetadata.json, JSONB)['_export_to'].contains('[{"HAL": true}]')
```

Reply:
I mean, it's totally not the point, but this bugged me.

Reply:
Done

Reply:
You can even do `contains({'HAL': True})`